app/admin/texts.rb

```diff
@@ -0,0 +1,25 @@
+ActiveAdmin.register Text do
+  permit_params :genre, :title, :content, :image
+
+  form do |f|
+    f.inputs "Texts" do
+      f.input :genre
+      f.input :title
+      f.input :content
+      f.input :image, :as => :file
```

Comment:
画像投稿機能を付けると指定する必要がなくなります。一旦こちらにしておいて下さい。

```suggestion
      f.input :image
```